packages/main/test/samples/ColorPalettePopover.sample.html

```diff
@@ -0,0 +1,106 @@
+<header>
+	<h2 class="control-header">ColorPalette</h2>
+</header>
+
+<div class="component-package">@ui5/webcomponents</div>
+
+<div class="control-tag">&lt;ui5-color-palette&gt;</div>
+
+<section>
+	<h3>Color Palette Popover with recent colors, default color and more colors features</h3>
+	<div class="snippet">
+		<ui5-button id="colorPaletteBtn" >Open ColorPalettePopover</ui5-button>
+		<ui5-color-palette-popover id="colorPalettePopover" show-recent-colors show-more-colors show-default-color default-color="green">
+			<ui5-color-palette-item value="pink"></ui5-color-palette-item>
+			<ui5-color-palette-item value="darkblue"></ui5-color-palette-item>
+			<ui5-color-palette-item value="#444444"></ui5-color-palette-item>
+			<ui5-color-palette-item value="rgb(0,200,0)"></ui5-color-palette-item>
+			<ui5-color-palette-item value="green"></ui5-color-palette-item>
+			<ui5-color-palette-item value="darkred"></ui5-color-palette-item>
+			<ui5-color-palette-item value="yellow"></ui5-color-palette-item>
+			<ui5-color-palette-item value="blue"></ui5-color-palette-item>
+			<ui5-color-palette-item value="cyan"></ui5-color-palette-item>
+			<ui5-color-palette-item value="orange"></ui5-color-palette-item>
+			<ui5-color-palette-item value="#5480e7"></ui5-color-palette-item>
+			<ui5-color-palette-item value="#ff6699"></ui5-color-palette-item>
+		</ui5-color-palette-popover>
+	</div>
+	<script>
+		colorPaletteBtn.addEventListener("click", (event) => {
```

Comment:
Only the code wrapped in &lt;pre> tag should be ES6. This code should be in ES5.